src/main/proto/marathon.proto

```diff
@@ -90,6 +90,10 @@ message ResourceRoles {
 }
 
 message MarathonTask {
+  message ReservationWithVolumes {
+    repeated string volumeIds = 1;
```

Comment:
full ack; will rename and change to underscore separation instead of camel humps